examples/classification/README.md

```diff
@@ -41,65 +42,48 @@ python classification.py mobilenet_v2 --batch_size 8 --num_cores 4
 Run with the `-h` flag to see all available models.
 
 ## Benchmarking Example
-`benchmark.py` is a script for benchmarking sparsified image classification model performance with DeepSparse. For a full list of options run `python benchmark.py -h`.
+`deepsparse.benchmark` is a utility for benchmarking sparsified image classification model performance with DeepSparse. For a full list of options run `deepsparse.benchmark -h`.
 To run a benchmark using the DeepSparse Engine with a moderately pruned ResNet model that uses all available CPU cores and batch size 1, run:
 
 ```bash
-python benchmark.py "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate"
+deepsparse.benchmark "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate"
 ```
 
-Example using own model and data:
+The model stub can be replaced with any [SparseZoo](http://sparsezoo.neuralmagic.com/) stub or filepath to ONNX model file.
+Example:
```

Comment:
Does ^ this word "Example:" need to be removed?

Reply:
Good catch thanks Jeannie!